src/components/action-popover/action-popover.component.tsx

```diff
@@ -72,7 +72,7 @@ export const ActionPopover = ({
 }: ActionPopoverProps) => {
   const l = useLocale();
   const [isOpen, setOpenState] = useState(false);
-  const [focusIndex, setFocusIndex] = useState(0);
+  const [focusIndex, setFocusIndex] = useState<number | undefined>(0);
```

Comment:
comment: same comment as above re changing the type

Reply:
Yep and I'll use the same answer as before as it's all linked 😆 

I'll double check this. I'm pretty sure I was getting the TS error because of the `firstMatch` and `nextMatch` could potentially be `undefined` in the action popover menu keyboard nav function.